static/js/modules/helpers.js

```diff
@@ -216,6 +215,41 @@ function zeroPad(container, item, appendee) {
   });
 }
 
+function amendmentVersion(data) {
+  if (data === true) {
+    return '<i class="icon-circle--check-outline--inline--left"></i>Current version';
+  }
+  else if (data === false) {
+    return '<i class="icon-circle--clock-reverse--inline--left"></i>Past version';
+  }
+  else {
+    return 'Version unknown';
+  }
+}
+
+function amendmentVersionDescription(row) {
+  var description = '';
+  var amendment_num = 1;
+
+  // because of messy data, do not show if not e-filing or null amendment indicator
+  if (row.means_filed === 'e-file' && row.amendment_indicator !== null) {
+    if (row.amendment_indicator === 'N') {
```

Comment:
`N` and `e-file` are variables returned from the API. How would I refactor that?